modules/core/src/shaderlib/project/viewport-uniforms.js

```diff
@@ -144,7 +165,13 @@ function calculateViewportUniforms({
   const coordinateZoom = viewport.zoom;
   assert(coordinateZoom >= 0);
 
-  const {projectionCenter, viewProjectionMatrix, cameraPos} = calculateMatrixAndOffset({
+  const {
```

Comment:
This code is getting ugly... Too hard to improve?

Reply:
How?